pkg/hwinfo/hwinfo_nonwindows.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/cloudradar-monitoring/cagent/pkg/common"
 )
 
-func isCommandAvailable(name string) bool {
-	cmd := exec.Command("/bin/sh", "-c", "command", "-v", name)
+func isDmidecodeAvailable() bool {
+	cmd := exec.Command("/bin/sh", "-c", "command -v dmidecode")
```

Comment:
Does this execute correctly? Arguments usually wants to be separated to exec.Command() in order for the shell to correctly parse the command invoked.

Reply:
yep.

old code was wrong actually:
next argument to `sh -c` should be 'the command to execute'.